input/definitions/multiple/StructureDefinition-cqf-expansionParameters.json

```diff
@@ -59,8 +55,8 @@
       "id" : "Extension",
       "path" : "Extension",
       "short" : "What expansion rules",
-      "definition" : "Specifies the expansion parameters that should be used to expand the ValueSet, or any ValueSets in the Library (or referenced by components of the Library).",
-      "comment" : "For a complete description of how these expansion rules impact the behavior of the expansion operation, see the [Expansion Rules](http://build.fhir.org/ig/HL7/crmi-ig/artifact-terminology-service.html#expansion-rules) topic.",
+      "definition" : "Specifies the expansion parameters that should be used to expand ValueSets in the Library or ImplementationGuide (or referenced by component artifacts of the Library or Implementation Guide).",
+      "comment" : "Expansion parameters specified in an ImplementationGuide override expansion parameters specified in dependency IGs, and can in turn be overridden by expansion parameters specified in downstream IGs. When this extension is used, it is intended to be binding (i.e. expansion SHALL take the expansion parameters into account, according to the Expansion Rules discussed in the CRMI Artifact Terminology Service [Expansion Rules](http://hl7.org/fhir/uv/crmi/1.0.0-snapshot/artifact-terminology-service.html#expansion-rules) topic.",

```

Comment:
How does overriding work? Are all Expansion parameters replace if any are specified in a downstream IG, or just those with the same "name"? If the latter, are multi-valued expansion parameters treated the same way as single-valued expansion parameters (i.e., all replace vs just adding additional values)?

Reply:
The intent is for a complete replacement, because otherwise there would be no way to change system-version of an code system specified in an upstream ig, since system-version is multi-cardinality.